src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
These "enums" beyond being a direct implementation from the spec do serve a useful purpose for us. It allows us to much more easily have a default `kind`.
I'm OK with moving all the "enums" into a single `tracks-enums.js` file and I could be convinced of moving them into their respective track files.